lib/salus/report.rb

```diff
@@ -222,9 +224,26 @@ def publish_report(directive)
       end
     end
 
+    def satisfies_filter?(directive)
+      return true if @report_filter == 'all'
```

Comment:
The ticket says this would be `'*'`, not `'all'`.  
I'm ok with either. @joshua-ostrom, does it need to be replaced with `'*'`?

Ticket:
```
--reports name:* # run all reports
```